presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConfig.java

```diff
@@ -977,4 +995,15 @@ public HiveConfig setProjectionPushdownEnabled(boolean projectionPushdownEnabled
         this.projectionPushdownEnabled = projectionPushdownEnabled;
         return this;
     }
+
+    @PostConstruct
+    public void init()
+    {
+        verifyDefaultValueForInsertExistingPartitionsBehaviorCouldBeAssigned();
+    }
+
+    public void verifyDefaultValueForInsertExistingPartitionsBehaviorCouldBeAssigned()
+    {
+        verifyAndGet(insertExistingPartitionsBehavior, immutablePartitions);
+    }
```

Comment:
```suggestion
    public void validate()
    {
                InsertExistingPartitionsBehavior.validate(...);
    }
```

Please add tests for that. See `io.prestosql.plugin.hive.TestHiveHadoop2Plugin#testGcsAccessTokenAndHiveCachingMutuallyExclusive`

Reply:
I've added additional function to describe in it's name why do we need @PostConstruct at all. You think it worth to keep code as simple as possible without this explanation?

Reply:
done